src/index.ts

```diff
@@ -1,3 +1,7 @@
-export * from './dbs';
-export * from './models';
+// export * from './Nedb/dbs';
```

Comment:
remove comments

Reply:
That was left there to showcase this is the only change you need to uncomment to make use of Nedb functions.
Anyway removing it :)